frontend/src/components/content/toolbar.tsx

```
@@ -23,54 +25,57 @@ const headingLevels = [
+ *
  * @param editor - TipTap Editor (ToolbarProps)
  * @returns JSX.Element
  */
 export default function Toolbar({ editor }: ToolbarProps) {
-  const [isDarkMode , setIsDarkMode] = useState(false);
+  const [isDarkMode, setIsDarkMode] = useState(false);
 
-  {/*ANCHOR - Fix this Code, Light Mode / Dark Mode is not Actively Working. */}
+  {
+    /*ANCHOR - Fix this Code, Light Mode / Dark Mode is not Actively Working. */
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)